modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/CppRestClientCodegen.java

```diff
@@ -55,7 +55,7 @@
 
     /**
      * Configures the type of generator.
-     * 
+     *
```

Comment:
Auto format changes?

Reply:
Yes, I have my editor set to removing trailing whitespace.

Reply:
Please keep that whitespace in

Reply:
Why is that?